packages/cdktf-cli/bin/cmds/output.ts

```diff
@@ -0,0 +1,51 @@
+import * as yargs from "yargs";
+import { config as cfg } from "@cdktf/provider-generator";
+import { requireHandlers } from "./helper/utilities";
+
+const config = cfg.readConfigSync();
+
+class Command implements yargs.CommandModule {
+  public readonly command = "output [stack] [OPTIONS]";
+  public readonly describe = "Prints the output of a stack";
+  public readonly aliases = ["outputs"];
+
+  public readonly builder = (args: yargs.Argv) =>
+    args
+      .positional("stack", {
+        desc: "Get outputs of stack which matches the given id only. Required when more than one stack is present in the app",
+        type: "string",
+      })
+      .option("app", {
+        default: config.app,
+        required: true,
+        desc: "Command to use in order to execute cdktf app",
+        alias: "a",
+      })
+      .option("output", {
+        default: config.output,
+        required: true,
+        desc: "Output directory",
```

Comment:
I know this flag existed before (for all commands requiring a synth) but in this context I'd love to see the description improved a bit (and maybe also copy that for the others) because it might be confused as the directory for the `outputs-file` option.
Maybe something along the lines of "output directory for the synthesized Terraform config"?